Sources/Public/CombineExtension.swift

```diff
@@ -63,8 +63,26 @@
                     Pretty.prettyPrint($0, option: option, to: &s)
                     _print(s, type: .output, terminator: "")
                 }
-            }, receiveCompletion: {
-                _print("receive \($0)", type: .completion)
+            }, receiveCompletion: { completion in
```

Comment:
Please add unit tests to:
https://github.com/YusukeHosonuma/SwiftPrettyPrint/blob/master/Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift

Reply:
added!